app/views/ssh_keys/_ssh_keys_tab.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="tab-pane" id="ssh_keys">
+  <table class="<%= table_css_classes 'table-fixed' %>">
+    <% if user.ssh_keys.any? %>
+    <thead>
+      <tr>
+        <th width="40%"><%= _('Name') %></th>
+        <th width="40%"><%= _('Fingerprint') %></th>
+        <th width="80px"><%= _('Length') %></th>
+        <th width="20%"><%= _('Created') %></th>
+        <th width="100px"><%= _('Actions') %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% user.ssh_keys.each do |key| %>
+        <%= render 'ssh_keys/ssh_key_row', :f => f, :key => key %>
+      <% end %>
+    </tbody>
+  <% else %>
+    <tbody>
+      <tr>
+        <td class="blank-slate-pf">
+            <div class="blank-slate-pf-icon">
+              <%= icon_text("key", "", :kind => "fa") %>
+            </div>
+            <h1><%= _('SSH keys') %></h1>
+            <p>
+              <%= _("You can add SSH public keys to a user in Foreman.") %><br />
+              <%= _("The keys can then be used in templates and are available in config management tools managed by Foreman.") %><br />
```

Comment:
nitpick, I think the 'then' here can be spared, and "config" -> "configuration", or "config.". Also "available for".

"The keys can be used in provisioning templates and are available for configuration management tools managed by Foreman."

cc @domcleal @GregSutcliffe if you can double check.. sorry